### PR TITLE
refactor(authentication-service): update urls and add scope for keycloak idp

### PR DESCRIPTION
**Pull request by Tyagi-Sunny**
update urls and add scope for keycloak idp

BREAKING CHANGE:
yes

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Intermediate change (work in progress)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

## Checklist:

- [ ] Performed a self-review of my own code
- [ ] npm test passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the style guide
- [ ] API Documentation in code was updated
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Surbhi-sharma1 — APPROVED** [review]

**Comment by yeshamavani** [comment]
@Tyagi-Sunny update the readme file saying that people using the older version can still use this by only updating the KEYCLOCK_HOST and appending '/auth/realms' to its existing value

**Comment by Tyagi-Sunny** [comment]
> @Tyagi-Sunny update the readme file saying that people using the older version can still use this by only updating the KEYCLOCK_HOST and appending '/auth/realms' to its existing value

added

File: services/authentication-service/README.md [hunk 1]
```diff
@@ -405,7 +405,7 @@ npm i @sourceloop/authentication-service
       <tr>
         <td>KEYCLOAK_HOST</td>
         <td>N</td>
-        <td>Hostname of the Keycloak instance</td>
+        <td>Hostname of the Keycloak instance. For all keycloak version below 17, user can still use this by only updating the KEYCLOCK_HOST and appending '/auth/realms' to its existing value</td>
         <td></td>
       </tr>
       <tr>
```
Comment (yeshamavani) [inline services/authentication-service/README.md:411]: what about the extra scope parameter that we are passing 
will that cause any issue for older versions of keyclock

Comment (Tyagi-Sunny) [reply services/authentication-service/README.md:411]: Adding a default scope: 'openid' is good practice, as some configurations expect this scope to be explicitly specified for OpenID Connect flows.The scope parameter should work across Keycloak versions without issue.

**Pull request merged**